backend/app/api/v1/items.py

```
@@ -0,0 +1,111 @@
+"""Item-related API routes."""
+
+from decimal import Decimal
+from typing import List
```

Ruff (0.12.2)

`typing.List` is deprecated, use `list` instead

(UP035)